handler/interceptors.go

```diff
@@ -102,15 +107,28 @@ type PaymentHandler interface {
 
 type rateLimitInterceptor struct {
 	rateLimiter rate.Limiter
+	subscribeToNotification *configuration_service.MessageBroadcaster
```

Comment:
I suggest renaming to `messageBroadcaster` as `subscribeToNotification` looks unclear for me.